library/ecdh.c

```diff
@@ -137,13 +137,20 @@ int mbedtls_ecdh_compute_shared( mbedtls_ecp_group *grp, mbedtls_mpi *z,
 
 static void ecdh_init_internal( mbedtls_ecdh_context_mbed *ctx )
```

Comment:
The code is ok, but the convention in the library is that an `init` function makes no assumption about the content of its argument — it's typically used on uninitialized memory. Please rename this function, e.g. to `ecdh_reset_internal`.